CHANGELOG.md

```diff
@@ -2,9 +2,39 @@
 
 ## Unreleased
 
-### Fix 
+### Add
+- **Opt-in support for muxed accounts.** This introduces `M...` addresses from [CAP-27](https://github.com/stellar/stellar-protocol/blob/master/core/cap-0027.md), which multiplex a Stellar `G...` address across IDs to eliminate the need for ad-hoc multiplexing via the Transaction.memo field (see the relevant [SEP-29](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0029.md) and [blog post](https://www.stellar.org/developers-blog/fixing-memo-less-payments) on the topic). The following operations now support muxed accounts ([#416](https://github.com/stellar/js-stellar-base/pull/416)):
```

Comment:
I would stick to mentioning SEP23 (unless raw XDR suport for muxed accounts wasn't there already).If you disagree (fair) at the very least I would mention SEP23

Reply:
To clarify, M-addresses are covered by SEP23, CAP27 introduces the XDR representation. Those are separate specifications.

Reply:
Good call, I completely overlooked the existence of SEP-23.

Reply:
Resolved in 857faad626da638a24282313b428e68a5b43833f.